### ISSUE TITLE
Install on virgin Debian: "Gdk-CRITICAL" error on startup

### ISSUE THREAD

**Issue by pandascrape** [bisq-network/support#896]
I am trying to install Bisq 1.5.9 on Debian 10 (Buster). It is a fresh Debian installation. The only two other apps I installed work fine.

- I run `sudo apt install ./Bisq-64bit-1.5.9.deb` and it installs correctly with no errors.
- When I run 'Bisq' I can see the Bisq process briefly appearing in linux list of processes if I run `top` and it creates the `~/.local/share/Bisq` directory so the process must have started.
- However, the app GUI never appears, and the bisq process disappears from the list of processes after a few seconds.

No error messages are given.

Bisq version: 1.5.9
Device or machine: Debian 10

**Comment by RififiCastorjunior** [comment]
Hi, Could you try this command `/opt/Bisq/Bisq`

That should lunch Bisq.

**Comment by pandascrape** [comment]
Unfortunately this does not work.

There is no `Bisq` directory under `/opt/` on the system.

**Comment by pandascrape** [comment]
When I execute with `/opt/Bisq/Bisq` I receive the following errors:

```
(Bisq:2316): Gdk-CRITICAL **: 09:02:48.357: gdk_x11_display_set_window_scale: assertion 'GDK_IS_X11_DISPLAY (display)' failed
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007fb006bf39fb, pid=2316, tid=2358
#
# JRE version: Java(TM) SE Runtime Environment (10.0.2+13) (build 10.0.2+13)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (10.0.2+13, mixed mode, tiered, compressed oops, g1 gc, linux-amd64)
# Problematic frame:
# C  [libX11.so.6+0x2e9fb]  XInternAtom+0x3b
#
# No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /tmp/hs_err_pid2316.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
Aborted
```

**Comment by pandascrape** [comment]
Hi again,

I have tried this on a different laptop with a totally fresh install of Debian 10 and get the same issue (so that's two laptops with fresh installs). The error file referred to is attached.
[hs_err_pid2316.log](https://github.com/bisq-network/support/files/6223151/hs_err_pid2316.log)


However when I install over an existing old version of Bisq (on a third machine running Debian 10), it works fine.

Wanting it on a clean machine though, so any help you can offer is much appreciated.

**Comment by pandascrape** [comment]
It appears a similar issue occurred in 2019: https://github.com/bisq-network/bisq/issues/2279 however it is not clear how this was resolved (to me anyway).

**Comment by pandascrape** [comment]
I have a temporary workaround, which is to force my Debian GNOME 3 to use X11 rather than Wayland by uncommenting the `WaylandEnable=false` line in the `/etc/gdm3/daemon.conf` file and rebooting the system.

This allows me to run Bisq, but because GNOME 3 (and many other desktops) are moving to Wayland this obviously needs to be addressed by the Bisq team otherwise new users will not be able to use it. Thanks for a great project, people!

**Comment by cd2357** [comment]
Fixed with v1.6.4 as per https://github.com/bisq-network/bisq/issues/2279#issuecomment-836277605

@leo816 can you please close this?
